webpack.config.babel.js

```diff
@@ -34,6 +35,12 @@ const plugins = [
     new webpack.ProvidePlugin({
         [options.jsxPragma]: path.join(__dirname, "utils", "element"),
     }),
+    new CopyWebpackPlugin([
+    {
+        from: "./extension-config",
+        to: "./"
```

Comment:
So this copies the config to the output directory? Would you mind adding a comment to that effect?

Reply:
It copies the the extension configuration and any extension assets to the build folder.